### PR TITLE
Update docs for version 9

### PR DESCRIPTION
**Pull request by LuizMoratelli**

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -15,11 +15,91 @@ let schema = [
 <SignUpFormParams schema>
 ```
 
-### Required
-For required changes. For now Required checks if the value is `""` to throw the error.
+### NoValidation
+Use for optional fields. NoValidation accepts any or no value
 ```reason
 let schema = [
```
Comment (fakenickels) [inline docs/schema.md]: this doesn't exist anymore right

File: docs/schema.md [hunk 2]
```diff
@@ -15,11 +15,91 @@ let schema = [
...
+];
+
+<SignUpFormParams schema>
+```
+
+### IntMin
+Use for integer that must have minimum value.
+```reason
+let schema = [
+  (`age, IntMin(18))
```
Comment (fakenickels) [inline docs/schema.md]: hmm none of these exist anymore

Comment (fakenickels) [reply docs/schema.md]: should be like we used with lenses.

File: docs/schema.md (hunk 1 above)
Comment (LuizMoratelli) [reply docs/schema.md]: Exactly, this existed in Reform_Validation no more in ReFormNext

**Comment by fakenickels** [comment]
I'm going to add the base for this so after you can update there

**Pull request closed without merging**